server/tests/lib/nl/date_parser_test.py

```diff
@@ -34,6 +34,8 @@ class TestDateParser(unittest.TestCase):
       ('Earthquakes in the last 5 years',
        [Date('before', 2022, 0, year_span=5)]),
       ('Flood in previous year', [Date('before', 2022, 0, year_span=1)]),
+      ('Population over the past decade',
```

Comment:
nit: would be nice to have some strings which do not trigger the date detector.

Reply:
done